templates/repo/clone_script.tmpl

```diff
@@ -17,7 +19,9 @@
 
 		const link = btn.getAttribute('data-link');
 		for (const el of document.getElementsByClassName('js-clone-url')) {
-			el[el.nodeName === 'INPUT' ? 'value' : 'textContent'] = link;
+			let attr = el.nodeName === 'INPUT' ? 'value' : 'textContent';
+			attr = el.getAttribute('data-attr') || attr;
+			el[attr] = `${el.getAttribute('data-prefix') || ''}${link}`;
```

Comment:
TBH, I think these lines are inlined too aggressively, not easy to read 😂

And `vscode://vscode.git/clone?url=` need to be appended with an escaped URL.

I think it could be clear by just writing: `getElementsByClassName('js-clone-url-vs')`  and set the href separately.